src/Aspire.Cli/Utils/InteractionUtils.cs

```diff
@@ -42,4 +43,10 @@ public static async Task<T> PromptForSelectionAsync<T>(string promptText, IEnume
 
         return await AnsiConsole.PromptAsync(prompt, cancellationToken);
     }
+
+    public static int DisplayIncompatibleVersionError(AppHostIncompatibleException ex)
+    {
+        AnsiConsole.MarkupLine($"[red bold]:thumbs_down:  The app host is not compatible. {ex.Message}[/]");
```

Comment:
Add something about upgrading the CLI or the app host. Do we know which one is older here?

Reply:
![image](https://github.com/user-attachments/assets/df0f6e7d-a194-4c59-977a-427a19590abf)

Updated.